packages/shared/src/components/modals/FeedFiltersModal/FeedFiltersIntroModalTags.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement, useState } from 'react';
+import { FeedFiltersIntroModalTagPill } from '../../utilities';
+
+type FeedFiltersIntroModalTagsProps = {
+  tag: string;
+  shouldBeCut: boolean;
+};
+
+const widths = ['w-[4.5rem]', 'w-24', 'w-[7.5rem]'];
+
+const getRandomWidth = () => {
+  const randomIndex = Math.floor(Math.random() * widths.length);
+  return widths[randomIndex];
+};
+
+const getWidth = (shouldBeCut: boolean) => {
+  if (shouldBeCut) {
+    return 'flex-1';
+  }
+  return getRandomWidth();
+};
```

Comment:
As this isn't reused and relatively small should we not perhaps combine them?

```suggestion
const getWidth = (shouldBeCut: boolean) => {
  if (shouldBeCut) {
    return 'flex-1';
  }
  
  const randomIndex = Math.floor(Math.random() * widths.length);
  return widths[randomIndex];
};
```